src/index.css

```diff
@@ -1,13 +1,237 @@
+
+html,
+body {
+  height: 100%;
+  position: relative;
+}
+
+html {
+  font-size: 125%;
+  font-family: 'Source Code Pro', monospace;
+  }
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  background-color: #000;
+}
+
+.mainContainer {
+  min-height: 100vh;
+  display: block;
+  position: relative;
+  padding-bottom: 100px;
+}
+
+.wrapper {
+  max-width: 1200px;
+  width: 80%;
+  margin: 0 auto;
+}
+
+button {
+  font-size: 0.9rem;
+  background-color: #242424 ;
+  color: #d1cde4;
+  padding: 10px;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+/* Header Styling */
+.headerContainer {
+  padding-top: 150px;
+}
+
+header {
+  background-image: url(./19849.jpg);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position: bottom;
+  height: 60vh;
+}
+
+h1 {
+  font-size: 2rem;
+  text-align: left;
+  color: white;
+}
+
+h1 span {
+  display: block;
+}
+
+header p {
+  color: white;
+  width: 55%;
+  line-height: 1.4;
+}
+
+/* Form Styling */
+form {
+  background-image: linear-gradient(-45deg,#fcda7e,#ff9733, #da4014);
+  background-size: 400%;
+  animation: gradient 15s ease infinite;
+  height: 150px;
+  width: 50%;
+  margin: -50px auto 10px auto;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  }
+  
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+
+/* Results Display Styling */
+.resultsContainer {
+  display: flex;
+  flex-wrap: wrap; 
+}
+
+.results {
+  width: calc((100% / 3) - 20px);
+  background-color: #d1cde4;
+  margin: 10px;
+  padding: 8px;
+}
+
+h2 {
+  font-size: 1rem;
+}
+
+.results img {
+  max-width: 100%;
+}
+
+.unliked {
+  color: red;
+}
+
+/* Search Again Button Styling */
+.searchAgain {
+  background: #d1cde4;
+  display: block;
+  margin-right: 10px;
+  margin-left: auto;
+  height: 50px
+}
+
+/* Pupup Styling */
+.popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.2);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.popupInner {
+  position: relative;
+  padding: 10%;
+  width: 50%;
+  background-color: #d1cde4;
+}
+
+.popupInner .closeButton {
+  position: absolute;
+  top: 16px;
+  right: 16px;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+/* Footer Styling */
+footer {
+  color: white;
+  position: absolute;
+  bottom: 0%;
+  width: 95%;
+  left: 50px;
 }
+
+footer a {
+  color: white;
+}
+
+/* Media Queries */
```

Comment:
ok, really love the media queries here for each page break - the attention to detail for every little thing that changes as the size of the screen changes is what separates good responsive design from great responsive design (well, that's my opinion anyway)